include/dynd/kernels/get_then_copy_kernel.hpp

```diff
@@ -11,7 +11,40 @@
 namespace dynd {
 namespace nd {
 
-  template <typename TypeType, const array &(TypeType::*Func)() const>
+template <typename TypeType, array (TypeType::*Func)() const>
+struct get_then_copy_kernel2 : base_kernel<get_then_copy_kernel2<TypeType, Func>, 0> {
```

Comment:
Does this completely replace the old `get_then_copy_kernel`? Why not make the changes to the original rather than having two kernels?